Userland/Utilities/cp.cpp

```diff
@@ -20,6 +20,7 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
     auto preserve = FileSystem::PreserveMode::Nothing;
     bool recursion_allowed = false;
     bool verbose = false;
+    bool interactive = false;
```

Comment:
This flag is actually never used in the code you added.

Reply:
Oh, yeah, you're correct, I completely forgot. That's embarrassing, let me fix it quickly :^)